CubicrootXYZ/matrix-reminder-and-calendar-bot
# Send available reactions

## [FEATURE]: Send available reactions

Attach often used reactions to the messages so the user knows which are available.

**Related Issues:**

none

### Testing

1. Add a new reminder 
2. Check the reactions the bot adds

### Dependencies

none

### Before merge checklist

- [x] Changes are tested (code is actually executed and proofed working)
- [x] Documentation reflects the changes if necessary
- [x] Documentation states the newest version number
- [x] Pipeline/Action is executed successfully
- [x] All `Todo`s in the code done

### After merge checklist (for code owners)

- [x] Version number increased if necessary 
- [x] Version number increased in documentation if necessary
- [x] Changes added to draft of next release
